### PR TITLE
Update ReadMe

### PR DESCRIPTION
**Pull request by SimNee**
Checking if Travis is working.
Changed the UI and the badges for readme

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,9 +1,37 @@
-= Address Book (Level 3)
+= ORGANice (Organ Transplant Manager)
```
Comment (WilliamRyank) [inline README.adoc]: just remove the `Organ Transplant Manage`? cos you will describte it below + we remove OTM in all of our code alr

File: README.adoc [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
+For a transplant to be successful, we need to have an organ that matches the receiver. +
+Since time is precious during transplant, we aim to provide an organ matching system to efficiently match organs from donors to the patients who need them.
+
+== Target Users
+
+This application is intended for hospital admin staff such as doctors and nurses.
+
+== Installation
+
+You can download the https://google.com/[jar] file and run it from your desktop
```
Comment (WilliamRyank) [inline README.adoc]: for jar file use our release link `https://github.com/AY1920S1-CS2103T-T13-1/main/releases`

File: README.adoc [hunk 3]
```diff
@@ -12,22 +40,25 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
+== Build Status
+
```
Comment (WilliamRyank) [inline README.adoc]: can remove this line,

File: README.adoc (hunk 1 above)
Comment (SimNee) [reply README.adoc]: Oh wait they revert back the changes. Ok can. Intellij and github different for me, no idea why

File: README.adoc [hunk 4]
```diff
@@ -12,22 +40,25 @@ ifndef::env-github[]
...
 * <<UserGuide#, User Guide>>
 * <<DeveloperGuide#, Developer Guide>>
-* <<LearningOutcomes#, Learning Outcomes>>
 * <<AboutUs#, About Us>>
 * <<ContactUs#, Contact Us>>
 
 == Acknowledgements
 
-* Some parts of this sample application were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
+* Some parts of Organice were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
```
Comment (WilliamRyank) [inline README.adoc]: ORGANice

File: README.adoc (hunk 2 above)
Comment (SimNee) [reply README.adoc]: Noted with thanks

File: README.adoc (hunk 1 above)
Comment (WilliamRyank) [reply README.adoc]: 🤔

Comment (WilliamRyank) [reply README.adoc]: maybe u havent push

File: README.adoc (hunk 3 above)
Comment (SimNee) [reply README.adoc]: I think its they auto add this one. Not sure why. Because my intellij dont have

File: README.adoc (hunk 4 above)
Comment (SimNee) [reply README.adoc]: Noted

File: README.adoc (hunk 3 above)
Comment (WilliamRyank) [reply README.adoc]: HAHAHAHA 🤔

Comment (SimNee) [reply README.adoc]: wait do you mean remove build status entirely?

**Pull request merged**